xlint

```diff
@@ -251,11 +251,13 @@ for template; do
 		done
 	fi
 
-	: ${LICENSE_LIST:=/usr/share/spdx/license.lst}
+	: "${LICENSE_LIST:=/usr/share/spdx/license.lst}"
 	if [ -f "${LICENSE_LIST}" ]; then
 		sed -n 's/license="\(.*\)"/\1/p' "$template" | tr , "\n" | while read -r l; do
 			if echo "$l" | grep -q 'custom:'; then
```

Comment:
Would be nicer if this just was a case..esac.

Reply:
done